nodes/config/ui_page.html

```diff
@@ -1,107 +1,256 @@
+                    { name: 'Desktop', px: 1024, cols: 12 }
+                ]
+
+                if (!this.breakpoints) {
+                    this.breakpoints = defaultBreakpoints
+                }
+
+                for (let i = 0; i < this.breakpoints.length; i++) {
+                    const breakpoint = this.breakpoints[i]
+                    breakpointsList.editableList('addItem', breakpoint)
+                }
+
+                // setup reset to defaults button
+                $('#node-config-breakpoints-reset').on('click', function (e) {
+                    e.preventDefault()
+                    breakpointsList.editableList('empty')
+                    for (let i = 0; i < defaultBreakpoints.length; i++) {
+                        const breakpoint = defaultBreakpoints[i]
+                        breakpointsList.editableList('addItem', breakpoint)
+                    }
+                })
+
+                // backwards compatibility - because NR can't do boolean values on dropdowns
+                if (this.visible === false || this.visible === 'false') {
+                    this.visible = false
+                    $('#node-config-input-visible').val('false')
+                } else {
+                    this.visible = true
+                    $('#node-config-input-visible').val('true')
+                }
+
+                // backwards compatibility
+                if (this.disabled === 'true' || this.disabled === true) {
+                    this.disabled = true
+                    $('#node-config-input-disabled').val('true')
+                } else {
+                    this.disabled = false
+                    $('#node-config-input-disabled').val('false')
+                }
+            },
+            oneditsave: function () {
+                const node = this
+                // convert string to boolean
+                const visible = $('#node-config-input-visible').val()
+                if (visible === 'true') {
+                    this.visible = true
+                } else {
+                    this.visible = false
+                }
+
+                // convert string to boolean
+                const disabled = $('#node-config-input-disabled').val()
+                if (disabled === 'true') {
+                    this.disabled = true
+                } else {
+                    this.disabled = false
+                }
+
+                // breakpoints
+                const breakpoints = $('#node-config-list-breakpoints').editableList('items')
+    
+                node.breakpoints = []
+                breakpoints.each(function (i) {
+                    const row = $(this)
+                    const breakpoint = {
+                        name: row.find('.node-input-breakpoint-name').val(),
+                        px: parseFloat(row.find('.node-input-breakpoint-px').val()),
+                        cols: parseFloat(row.find('.node-input-breakpoint-cols').val())
```

Comment:
```suggestion
                        px: row.find('.node-input-breakpoint-px').val(),
                        cols: row.find('.node-input-breakpoint-cols').val()
```

`parseFloat` will try to evaluate env var overrrides like `${MY_BRK_POINT}` and will result in `NaN` being saved as the value.

In short, sanitisation and defaults should probably be made server side (if at all)

Reply:
exec decision: removing parseFloat in this 1st iteration